PagerDuty/pagerduty.30s.rb

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/ruby
+# <bitbar.title>PagerDuty</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.a uthor>Allan Frese</bitbar.author>
+# <bitbar.author.github>frese</bitbar.author.github>
+# <bitbar.desc>Shows current pagerduty alert status.</bitbar.desc>
+# <bitbar.dependencies>ruby (httparty gem)</bitbar.dependencies>
+
+require "json"
+require "pp"
+require "optparse"
+require "httparty"
+require "date"
+
+#--------------------------------------------------------------------
+# Create a configuration file ".pagerduty" in your HOME directory
+# TOKEN: <pagerduty-token>
+# DOMAIN: <your-pagerduty-domain>
+# USERID: <your-pager-duty-user-id>
+#--------------------------------------------------------------------
+
+class PagerDuty
+
+    def main
+        begin
+            # Init some defaults
+            $verbose = false
+            $id      = nil
+            $command = "GET"
+            $color   = Hash.new
+            begin
+                $config  = YAML.load(File.read("#{ENV['HOME']}/.pagerduty"))
```

Comment:
Sorry... actually I meant just standard variables at the top of the file, which consumers of the plugin just edit. Have a look at, for example, [subreedit_links.rbL6-19](https://github.com/matryer/bitbar-plugins/blob/475e51caaa0193cc0a749c768b71173fe5cc74d6/Web/Reddit/subreddit_links.rb#L6-L19) which lets you set a list of reddits to subscribe to